trafgen_parser.y

```diff
@@ -147,7 +147,7 @@ static inline void __setup_new_counter(struct counter *c, uint8_t start,
 {
 	c->min = start;
 	c->max = stop;
-	c->inc = stepping;
+	c->inc = (type == TYPE_INC) ? stepping : -stepping;
```

Comment:
I think this won't work as is because c->inc is an uint8_t currently. But changing the type to int8_t (or int16_t if we want to allow increments/decrements larger/smaller than 127/-127) should be fine IMO.

Reply:
it works for 0 <= stepping <= 127: in such interval ((uint8_t) -stepping) == -((int)stepping).

To properly support larger stepping, I think we could as well change c->inc  type to int, which should let the compiler generate faster code

Reply:
Changing c->inc type to int sound good to me.

Reply:
oks, I refreshed the pull request, with the above change.

replying here, too, since up to sometime ago github did not send notification for pull request update